portfolio/src/main/webapp/art-gallery.js

```diff
@@ -0,0 +1,96 @@
+let imagesArray = [
```

Comment:
what do you mean by this? I created an array of the images so that I can reference the names and pull up the appropriate image when going through the gallery slideshow

Reply:
Ah sorry a lot of changes hard to track all of it at once...

If you wanted you could basically just have a

```
<div id="gallery"></div>
```

and then in this JS when you're loading the page you could insert a div, add a class, and set the img src all at once so then in your html file you don't have to manually have added all of these divs. Completely optional, as what you have works